polytope/polytope.py

```diff
@@ -570,6 +564,18 @@ def _rotate(polyreg, i=None, j=None, u=None, v=None, theta=None, R=None):
     return R
 
 
+def _Givens_rotation_matrix(i, j, theta, N):
```

Comment:
Oh, but that could be addressed by opening multiple views of the same file, positioned at different locations. Using [Atom](https://atom.io), these views could be different tabs side by side, or even different tabs in the same pane. Also, the [`minimap` package](https://atom.io/packages/minimap) together with packages that show changed lines using `git` are quite convenient for accelerating navigation. For long text files (like `polytope`), editing a file while open in 2 or 3 tabs can exhibit slight lag, but that is a minor issue, and can be mitigated by switching to a different tab.